Test/FMI3/fmi3_import_variable_test.cpp

```diff
@@ -118,7 +118,7 @@ static void test_binary_all_attrs(fmi3_import_t* xml) {
     REQUIRE(fmi3_import_get_variable_vr(v) == 4);
     REQUIRE(fmi3_import_get_causality(v) == fmi3_causality_enu_output);
     REQUIRE(fmi3_import_get_variability(v) == fmi3_variability_enu_discrete);
-    REQUIRE(fmi3_import_get_initial(v) == fmi3_initial_enu_exact);
+    REQUIRE(fmi3_import_get_initial(v) == fmi3_initial_enu_calculated);
```

Comment:
Was giving error because exact requires a start value, so I just changed to calculated